2025-08-03 - Mastering GitHub Copilot customization with Copilot-instructions.md

```
@@ -0,0 +1,162 @@
+
+### Configuration in VS Code
+
+If you don’t see these settings, make sure you have the latest Copilot extension installed.
+
+To enable Copilot customization in Visual Studio Code, you need to ensure the following settings are configured:
+
+1\. Open your VS Code settings (File > Preferences > Settings).
+
+2\. Search for “Copilot” and ensure the following settings are enabled:
+
+![](https://miro.medium.com/v2/resize:fit:1400/format:webp/1*L0Vv0AxeagWmZ7iqtWJZNQ.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)